tools/playground/samples.js

```diff
@@ -18,7 +18,8 @@ App.components = { Greeter };
 // Application setup
 // Note that the xml templates are injected into the global TEMPLATES variable.
 const qweb = new owl.QWeb({ templates: TEMPLATES});
-const app = new App({ qweb });
+owl.config.defaultEnv = { qweb };
```

Comment:
we want to automatically register all templates in the iframe, so we do not need to do this